Interop/Runtime/src/native/kotlin/kotlinx/cinterop/Pinning.kt

```diff
@@ -79,3 +92,15 @@ private external fun getAddressOfElement(array: Any, index: Int): COpaquePointer
 @Suppress("NOTHING_TO_INLINE")
 private inline fun <P : CVariable> Pinned<*>.addressOfElement(index: Int): CPointer<P> =
         getAddressOfElement(this.get(), index).reinterpret()
+
+@SymbolName("Kotlin_Arrays_getAddressOfElement")
```

Comment:
Maybe specialized versions here and above?

Reply:
I don't believe specialization would have any measurable effect here.